### DISCUSSION TITLE
Exclude properties for comparing

### DISCUSSION THREAD

**Discussion by djordjedjukic** [fluentassertions/fluentassertions#1519] [Q&A] [answered]
I am trying to use exclude option, but it doesn't work for me. 

I would expect that code bellow returns that objects are equal if only `PropertyOne` is different, but this doesn't work as expected.

`objectOne.Should().BeEquivalentTo(objectTwo, options => options.Excluding(x => x.PropertyOne));`

Is there some issue with this code?
Thank you

**Comment by jnyrup** [comment, marked as answer]
Do you have a complete example to share?
The code seems fine, but there could be several reasons/explanations to why it doesn't work as expected.

My immediate guess though is that `objectOne` and `objectTwo` are considered to be value types, e.g. if they override `Equals(object)`.

**Reply by djordjedjukic** [reply]
Oh, `objectOne` and `objectTwo` are C# records. Maybe that is the issue.

**Reply by jnyrup** [reply]
Yes, that's the cause.
See #1451 for discussions and workarounds for this.
